runtime/vm/ClassInitialization.cpp

```diff
@@ -157,11 +157,14 @@ performVerification(J9VMThread *currentThread, J9Class *clazz)
 		/* See if this class should be verified:
 		 *
 		 * - Do not verify any class created by sun.misc.Unsafe
+		 * - Do not verify hidden classes
 		 * - Do not verify any class which is marked for exclusion in the optional flags
 		 * - Verify every class whose bytecodes have been modified
 		 * - Do not verify bootstrap classes if the appropriate runtime flag is set
 		 */
-		if (!J9CLASS_IS_EXEMPT_FROM_VALIDATION(clazz) && J9_ARE_NO_BITS_SET(romClass->optionalFlags, J9_ROMCLASS_OPTINFO_VERIFY_EXCLUDE)) {
+		if (((!J9CLASS_IS_EXEMPT_FROM_VALIDATION(clazz)) && J9_ARE_NO_BITS_SET(romClass->extraModifiers, J9AccClassHiddenClass))
```

Comment:
Why does the EXEMPT macro allow hidden classes only to have them excluded here? Seems confusing.

Reply:
If I allow J9AccClassHiddenClass go through this, we will eventually go into https://github.com/eclipse/openj9/blob/master/runtime/bcverify/vrfyhelp.c#L957. As hidden class is not in the hashtable, we won't be find the ramclass, causing crash later inside the verifier.